modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -934,6 +934,11 @@ return view.extend({
 				o = ss.taboption('general', CBIWifiFrequencyValue, '_freq', '<br />' + _('Operating frequency'));
 				o.ucisection = s.section;
 
+				o = ss.taboption('general', form.Value, 'channels', _('Limit auto scan channels'),
+					_('Define ranges or individual channels e.g. 100 140 34-100'));
+				o.datatype = 'uinteger';
```

Comment:
if the datatype check is `uinteger`, ranges won't validate. Are ranges actually possible?

Reply:
Probably, but we are dealing with 2g, 5g ranges so could do 1-165?

Reply:
Up for any example or suggestions on this?